internal/volumes/mount.go

```diff
@@ -203,10 +203,19 @@ func (s *LinuxMountService) FormatDisk(disk string, fstype string) error {
 		_, _, err := command("mkfs.ext4", "-F", "-m0", disk)
 		return err
 	case "xfs":
-		_, _, err := command(
-			"mkfs.xfs",
-			"-i", "nrext64=0", // Compatibility with kernel that do not support nrext64
-			disk)
+		xfsConfigPath, err := GetXFSConfigPath()
+		if err == nil {
+			_, _, err = command(
+				"mkfs.xfs", "-f", "-c", fmt.Sprintf("options=%s", xfsConfigPath), disk)
+		} else {
+			// Fallback
+			// Default flags extracted from /usr/share/xfsprogs/mkfs/lts_4.19.conf
+			// for maximum backwards compatibility with older kernels
+			// Also see: https://man.archlinux.org/man/xfs_admin.8.en#O
+			// This document mentions the flags and their minimal supported kernel version
+			_, _, err = command(
+				"mkfs.xfs", "-i", "nrext64=0", "-m", "bigtime=0", "-m", "inobtcount=0", disk)
```

Comment:
Should we allow the users to set a cluster wide supported kernel version?

For example, if a node has a specific kernel version, creates and formats the volume, but then the volumes then moves to a different node with an older kernel version. Maybe having a cluster wide version can be handy for those use cases?

Reply:
So we:
1. Use the cluster wide configured kernel version
2. Use the current nodes kernel version as hopefully he has uniform cluster

Reply:
I wonder if we should allow the users to pass/point to their own XFS config file? If we do this, we might simply want to allow users to pass their own `mkfs` arguments, what do you think?

Currently, we rely on the XFS configs from the alpine distribution, I am not sure, but maybe there are some difference between distros that the user want to handle?

Maybe having the kernel version parsing and XFS config magic behind a feature flag is better?

What do you think about:
1. If provided, user xfs config (or mkfs args ?)
2. If provided, user xfs min kernel version 
3. Fallback to the oldest non EOL linux kernel xfs settings

Reply:
I agree on 1. and 2.

(3.) For a user who just wants to create a cluster of homogeneous nodes running the same Linux kernel, it may be simpler to rely on the `mkfs.xfs` default arguments instead of opting for the oldest supported configuration. As this way the user gets the latest XFS features.

Reply:
After discussing with @jooola we came with the following possible features:
1. User can pass args to `mkfs.xfs`
2. User can configure cluster wide min kernel version
3. User can set boolean to automatically choose the mkfs args based on the kernel version -> can only be used in homogeneous clusters
4. Default: Use the oldest non EOL linux kernel xfs args